exporter/coralogixexporter/exporter.go

```diff
@@ -30,12 +31,16 @@ type coralogixExporter struct {
 }
 
 // NewCoralogixExporter by Coralogix
-func newCoralogixExporter(cfg *Config, params component.ExporterCreateSettings) *coralogixExporter {
+func newCoralogixExporter(cfg *Config, params component.ExporterCreateSettings) (*coralogixExporter, error) {
+	if cfg.Endpoint == "" || cfg.Endpoint == "https://" || cfg.Endpoint == "http://" {
+		return nil, errors.New("coralogix exporter config requires an Endpoint")
+	}
```

Comment:
This is checked at validation so no need to check it here too

Reply:
Same comment as https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11065#discussion_r908139280